### PR TITLE
Cd relnotes 805xx cds 78504 v2

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request

This note supersedes the original CD 805xx release-notes PR which I had to close due to numerous merge-conflict issues:

https://github.com/harness/developer-hub/pull/3197

Preview links
- [NG Latest: Version 805xx](https://64fa7fe6c3c30c10919de26b--harness-developer.netlify.app/release-notes/continuous-delivery#latest-version-805xx)
- [CG Latest: Version 805xx](https://64fa7fe6c3c30c10919de26b--harness-developer.netlify.app/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes#latest-version-805xx)

### CODE REVIEW

File: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md [hunk 1]
```diff
@@ -16,7 +16,30 @@ For Harness on-prem releases, go to [Harness Self-Managed Enterprise Edition Rel
...
+
+### Early access features
+
+This release does not include early access features.
+
+### Fixed issues
+
+* Fixed an issue where using multiple HTTP Helm chart repositories could lead to an increase in CPU utilization on the delegate due to background validation tasks for the Harness HTTP Helm Repo connector. This was caused by running Helm repository updates during the validation tasks. (CDS-76433, ZD-48363)
+
+  This item requires Harness Delegate version 805xx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```
Comment (brian-f-harness) [inline docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md:34]:
```suggestion
  This item requires Harness Delegate version 80505. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

File: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md [hunk 2]
```diff
@@ -16,7 +16,30 @@ For Harness on-prem releases, go to [Harness Self-Managed Enterprise Edition Rel
...
+
+* Fixed a Nexus artifact issue where a fetch timed out when a single group contained more than 50 artifacts. (CDS-73884, ZD-45052, ZD-47206)
```
Comment (brian-f-harness) [inline docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md:36]:
```suggestion
* Fixed a Nexus artifact issue where a fetch timed out when a single group contained more than 50 artifacts. (CDS-73884, ZD-45052, ZD-47206)

   This item requires Harness Delegate version 80505. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

File: release-notes/continuous-delivery.md [hunk 3]
```diff
@@ -34,18 +34,97 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+* API call logs summary now includes details such as response, size, duration, HTTP verb, and response code. (OIP-767)
+
+* Fixed an issue that resulted in Null Pointer Exceptions when running a pipeline manually with a  `<+trigger.connectorRef>` expression. This expression gets its data from the trigger payload. With this fix, the pipeline correctly handles the case where the trigger payload is null.  (CDS-77736, ZD-49685, ZD-49720, ZD-49722)
+
+  This item requires Harness Delegate version 805xx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
+
+*  Fixed an intermittent issue where Helm deployment pipelines would report the Helm repository as not found.  (CDS-76919)
+
+  This item requires Harness Delegate version 805xx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:99]:
```suggestion
  This item requires Harness Delegate version 80505. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

File: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md (hunk 2 above)
Comment (brian-f-harness) [reply docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md:36]: @douglas-j-bothwell I think this is for NG, not CG

https://harness.atlassian.net/wiki/spaces/PD/pages/21482799105/NG+Release+SAAS+August+29th+2023+for+805xx-+Build

**Review by brian-f-harness — DISMISSED** [review]

**Comment by deepakbolangady** [comment]
The release notes have been peer-reviewed already. Just did some housekeeping in the last two commits. Forcing a merge.

**Pull request merged**